dao2caom2/telescopes.py

```diff
@@ -166,15 +166,21 @@ def update(self, observation, file_info, clients):
                         cc.undo_astropy_cdfix_call(chunk, time_delta)
 
                         if self._storage_name.file_name.startswith('d'):
-                            if plane.data_product_type == DataProductType.SPECTRUM:
+                            if (
```

Comment:
I'm thinking we should move to 120-wide lines to increase readability in cases like this.

Reply:
Two questions:
- change all repositories?
- will there be one commit with all the line length changes in it, to master?